src/s3ql/parse_args.py

```diff
@@ -182,6 +182,12 @@ def add_storage_url(self):
                           type=storage_url_type,
                           help='Storage URL of the backend that contains the file system')
 
+    def add_oauth(self):
+        self.add_argument("--oauth_type", metavar='<oauth_type>',default="google-storage",type=oauth_type,
```

Comment:
I think it's better to instead require the user to pass a storage url, and infer the correct backend from that. At some point we may want to generate tokens that are specific to the given bucket (or even prefix).

Reply:
I don't know if I understand, what you checked in the code is the parameter for choice oauth_type storage/drive when generating user/pass for your specific google drive/storage account.

Reply:
Yes. Drop that parameter, and require the user to specify the storage URL. Then you can look at the prefix (`gs://` or `gdrive://`) to determine whether the token is for Google Drive or Google Storage.